secretmanager/access-secret.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+- Init:
+    assign:
+      - secret_id: api_key
+      - version: latest
+- Get Project ID:
```

Comment:
No need for a full call to get project id, simply assign to a variable:

- project_id: ${sys.get_env("GOOGLE_CLOUD_PROJECT_ID")}